piptools/scripts/compile.py

```diff
@@ -46,6 +46,8 @@ class PipCommand(pip.basecommand.Command):
               help="Add header to generated file")
 @click.option('--index/--no-index', is_flag=True, default=True,
               help="Add index URL to generated file")
+@click.option('--emit-trusted-host/--no-trusted-host', is_flag=True, default=True,
```

Comment:
To avoid confusion, the --no flag should provide symmetry to the positive version: --no-emit-trusted-host.